tests/Fixtures/project/src/Application/CircularyWeaved.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Go\Tests\TestProject\Application;
+
+use Go\Tests\TestProject\Annotation as Aop;
+
+class CircularyWeaved
```

Comment:
This is the class that is weaved twice. First is weaved by LoggingAspect, because it is required for CircularWeavingAspect. Then it is matched by CircularWeavingAspect and that is where it fails.